lib/id3c/cli/redcap.py

```diff
@@ -289,7 +289,7 @@ def _fetch_records(self, parameters: dict) -> Iterator['Record']:
         return (Record(self, r) for r in self._fetch("record", parameters))
 
 
-    def update_records(self, records: List[Dict[str, str]]) -> int:
+    def update_records(self, records: List[Dict[str, str]], date_format: str = "YMD") -> int:
```

Comment:
This is where an enum could be handy, to know what the possible values are.

Reply:
Nod. I know there's a range of preferences on when to use enums, varying between "always" and "never". I'm somewhere in between; where exactly varies between languages too. An enum seemed like overkill to me here (as opposed to say, `InstrumentStatus`, which felt appropriate when I added it), but I haven't tried to articulate why. We can discuss more in Slack if you want.

As far as knowing what the possible values are: they're documented in the method's docstring and also validated at runtime with an assert. An enum _would_ add a small amount of compile-time safety.